README.md

```diff
@@ -1,5 +1,6 @@
 # Todo List
 
+In this project, we work more with classes and objects. And the introduction of a new lesson about OOP. The lesson shows the concepts of Single Responsibility and Tightly Coupled Objects.
 
 
```

Comment:
- [ ] The usage information in the README is incomplete. Please include information about the npm installation, and how to actually run the app locally using the package.json scripts like `npm start`.